### PR TITLE
Add "edit" button to column tag editor window

### PR DESCRIPTION
**Pull request by afontenot**
Instead of forcing "inline" editing in the Tag List Editor, which can be confusing, add a button that edits the selected tag pattern.

### CODE REVIEW

File: quodlibet/qltk/icons.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ class Icons(str):
     INSERT_IMAGE = "insert-image"
     INSERT_TEXT = "insert-text"
     LIST_ADD = "list-add"  # "_Add"
+    LIST_EDIT = "list-edit"  # "_Edit"
```
[the remark this answers is not in the document]
Comment (afontenot) [reply quodlibet/qltk/icons.py:84]: Oops, I meant to look this up and forgot about it. Unfortunately there seems to be no [Freedesktop Spec](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html) icon that covers the relevant action here. Looking at a variety of icon themes, it seems `document-edit` might be appropriate and more widely supported than `edit-rename`. For example, `edit-rename` isn't in Adwaita. Not sure what icon theme you're using but this choice might not work for most Gnome users.

[the remark this answers is not in the document]
Comment (declension) [reply quodlibet/qltk/icons.py:84]: OK `document-edit` works renders the same for me anyway

[the remark this answers is not in the document]
Comment (afontenot) [reply quodlibet/qltk/icons.py]: Fixed

**Comment by declension** [comment]
Thanks

**Pull request merged**